Scripts/BrightSign_Scripts/bsext_init

```diff
@@ -0,0 +1,13 @@
+#!/bin/sh
+
+PIDFILE=/var/run/haptic-server.pid
```

Comment:
We probably want to rename this pid file to touchfree, we don't have a haptic server any more